src/user/formats/hal.ts

```diff
@@ -280,6 +284,9 @@ export function newUserResult(user: User, password: string|null, identities: Pri
       'me': identities.map( identity => (
         { href: identity.uri, title: user.nickname ?? undefined }
       )),
+      'userinfo': {
+        href: `${user.href}/userinfo`,
+      },
```

Comment:
Also not needed. This implies a brand new route / endpoint but the new data simply exists on the user resource.

Reply:
yeah I noticed actually this endpoint doesnt exist... but it's in fact at `http://localhost:8531/userinfo`?

Reply:
That endpoint is a OICD-specific endpoint and returns information about the currently logged in user. The format is also slightly different.

Reply:
yeah I noticed... wasn't clear to me whether this was an endpoint intended for returning web-standard-complicant userinfo  JSON to front end only for a logged in user or otherwise.